### PR TITLE
Schema update for Table

### PR DESCRIPTION
**Pull request by dclaux**
# Related Issue

Fixes https://github.com/microsoft/AdaptiveCards/issues/5728

# Description

Adds the necessary declaration for the new Table element to the 1.5 schema.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/AdaptiveCards/pull/5855)

### CODE REVIEW

File: schemas/src/elements/Table.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+    "extends": "Element",
+    "description": "Provides a way to display data in a tabular form.",
+    "version": "1.5",
+    "properties": {
+        "columns": {
+            "type": "TableColumnDefinition[]",
+            "description": "Defines the number of columns in the table, their sizes ans more."
```
Comment (paulcam206) [inline schemas/src/elements/Table.json:9]: typo

```suggestion
            "description": "Defines the number of columns in the table, their sizes, and more."
```

**Review by paulcam206 — APPROVED** [review]

File: schemas/src/elements/TableCell.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/AdaptiveCards/6f39aedce45864ae1067ed44a5551dc973790bb5/source/nodejs/typed-schema/schema/lib/Type.json",
+    "description": "Represents a cell within a row of a Table element.",
+    "extends": "Container",
```
Comment (golddove) [inline schemas/src/elements/TableCell.json:4]: Container defines VerticalContentAlignment to be a required property with default "Top". This should now be changed to optional with no default, right? Same for horizontal alignment on TextBlock.

**Review by golddove — CHANGES_REQUESTED** [review]

File: schemas/src/elements/TableCell.json (hunk 2 above)
Comment (dclaux) [reply schemas/src/elements/TableCell.json:4]: Ah correct, will change

Comment (dclaux) [reply schemas/src/elements/TableCell.json:4]: Fixed

**Review by golddove — APPROVED** [review]

**Pull request merged**